provider/cpu_utils.py

```diff
@@ -98,8 +100,21 @@ def check_if_vm_vcpu_topology_match(session, os_type, cpuinfo):
                               " returned by guest:\n%s", out)
                 return False
 
+    threads_matched = cpuinfo.threads == threads
+    if devices and os_type == 'windows':
+        # Until QEMU 8.1 there was a different behaviour for thread count in case
+        # of Windows guests. It represented number of threads per single core, not
+        # the total number of threads available for all cores in socket. Therefore
+        # we disable check for older QEMU versions and adjust for newer versions.
+        if devices.qemu_version in VersionInterval("[, 8.0.0)"):
```

Comment:
Seems you agree but may missed to update the version ?   IMO, it is `(, 8.1.0)`

Reply:
Thanks for highlighting this. Yes, I pushed only the change reintroducing the Windows check. I corrected the version now (I used square brackets, but the meaning is the same).

Reply:
It is not related with the start or ( or [. It is about the end, the difference with 8.0.0 and 8.1.0. Use 8.1.0) is must IMO. Any 8.1.0-*** will be skipped.

Reply:
@yanan-fu Sorry, it took me a while! I got it now. 

So if I keept `"[, 8.0.0]"`, it would mean for example `8.0.1` will be excluded. Let me fix that :+1: